MSP_TASK

Mohul44 opened the pull request:
Do comment out my database values by yours.

Mohul44 committed:
imagefield

Mohul44 committed:
https://github.com/Mohul44/MSP-Recruitment-Web-Portal

gargvaibhav64 commented:
Please rename the 'static' directory back to original and only present changes which are necessary. If you need to access the static files, use 'Django's static url'. You can look into Django's documentation for more details "https://docs.djangoproject.com/en/2.2/howto/static-files/".

Mohul44 commented:
Hey,
I have not changed the 'static' directory I have just added a media folder for images and media files.
Also, I have added media_url and media_root which was necessary.

Mohul44 committed:
https://github.com/Mohul44/MSP-Recruitment-Web-Portal
